packages/design-system/src/components/web-components/ds-text-field/ds-text-field.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { action } from '@storybook/addon-actions';
+import { useEffect } from 'react';
+import { webComponentDecorator } from '../storybook';
+import './ds-text-field';
+
+export default {
+  title: 'Web Components/ds-text-field',
+  argTypes: {
+    'class-name': {
+      description: 'Additional classes to be added to the root button element.',
+      control: 'text',
+    },
+    'default-value': {
+      description:
+        'Sets the initial selected state. Using this attribute instead of `value` means that the select will maintain its own internal selection state.',
+      control: 'text',
+    },
+    disabled: { description: 'Disables the entire field.', control: 'boolean' },
+    'error-message': {
+      description: 'Enable the error state by providing an error message',
+      control: 'text',
+    },
+    'error-id': {
+      description:
+        'The ID of the error message applied to this field. If none is provided, the id will be derived from the `root-id` attribute.',
+      control: 'text',
+    },
+    'error-placement': {
+      description: 'Location of the error message relative to the field input',
+      options: [undefined, 'top', 'bottom'],
+      control: { type: 'radio' },
+    },
+    'field-class-name': {
+      description: 'Additional classes to be added to the dropdown button element',
+      control: 'text',
+    },
+    hint: {
+      description: 'Hint text or HTML',
+      control: 'text',
+    },
+    'hint-id': {
+      description: 'The ID of the hint element',
+      control: 'text',
+    },
+    inversed: {
+      description: 'Set to "true" to apply the "inverse" theme',
+      control: 'boolean',
+    },
+    label: {
+      description: 'Label text or HTML.',
+      control: 'text',
+    },
+    'label-class-name': {
+      description: 'Additional classes to be added to the field label',
+      control: 'text',
+    },
+    'label-id': {
+      description:
+        "A unique `id` to be used on the field label. If one isn't provided, a unique ID will be generated.",
+      control: 'text',
+    },
+    'label-mask': {
+      description:
+        'Applies date format masking to the input value entered and renders to a text field above the input. Passing `true` to `valueOnly` will return just the formatted value entered.',
+      options: [undefined, 'phone', 'zip', 'ssn', 'currency'],
+      control: { type: 'radio' },
+    },
+    name: {
+      description:
+        'The `name` is applied to a hidden select element that holds the selected value for the purpose of native HTML form support',
```

Comment:
The name value gets set both on the parent custom element and the child input tag.

Reply:
That's the same as some of our other web components. Do you see that being a problem?

Reply:
Updating the description here too

Reply:
I don't see it being a problem, just more flagging for the description. Pretty sure `name` only gets used on form submission, but also could be wrong.